internal/proto/buf.yaml

```diff
@@ -0,0 +1,14 @@
+version: v1
+name: buf.build/conduitio/conduit-processor-sdk
+deps:
+  - buf.build/haris/conduit-commons:v0.1.3
```

Comment:
Needs to change once https://github.com/ConduitIO/conduit-commons/pull/7 is merged.

Reply:
Updated.